src/stravalib/model.py

```diff
@@ -401,6 +402,11 @@ class BoundClientEntity(BaseModel):
 
 
 class RelaxedActivityType(ActivityType):
+    """This object supports allowing an array of Literal values to be used for
```

Comment:
Any classes that are missing docstrings return a warning when building the docs. i'm going through and adding docstrings as needed. This will NOT fix the recursion issue but will fix the warnings (and is more user friendly). so it's worth doing.